lib/group_site_manager.py

```diff
@@ -434,6 +435,8 @@ def _upload_to_group(self, numbers: List[str], group: str) -> None:
           self._upload_oaks(numbers)
         elif group == 'dtmd':
           self._upload_dtmd(numbers)
+        elif group == 'embdeals':
+          return self._upload_emb(numbers)
```

Comment:
let's remove the `return` here so it falls through and prints the statement on lines 440/443

Reply:
removed